unimport/unused.py

```diff
@@ -1,5 +1,238 @@
+import token
+
+from brm import NoLineTransposer, TokenTransformer, pattern
+
 from unimport.dedect import DetectUnusedImport
 
+dot_name = "(name( dot name)*)"
+newline_group = "(nl|newline)"
+
+
+class ParseResult:
+    def __init__(self, commas, modules, newlines):
+        self.commas = commas
+        self.modules = modules
+        self.newlines = newlines
+
+
+class ImportTokenTransformer(TokenTransformer):
+    # You can find the original example from:
```

Comment:
BRM license states you should keep copyright notice, not just a link 

https://github.com/isidentical/BRM/blob/dc54d39d824d8a8e9cd7fe666ad1156fea1775fe/LICENSE.txt#L9